api/src/functionalTest/java/uk/gov/hmcts/reform/divorce/caseapi/SaveAndSignOutNotificationTest.java

```diff
@@ -32,13 +37,20 @@
     @Value("${test-url}")
     private String testUrl;
 
+    @Value("${idam.s2s-auth.url}")
+    private String s2sUrl;
```

Comment:
At some point we might need to move all this out to some kind of helper to avoid boilerplate, but it's fine for now